tap_ordway/schemas/debit_memo.json

```diff
@@ -0,0 +1,102 @@
+{
+  "type": ["null", "object"],
+  "additionalProperties": false,
+  "required": ["company_id", "debit_memo_id"],
+  "properties": {
+    "debit_memo_id": {
+      "type": "string"
+    },
+    "customer_id": {  
+      "type": "string"
+    },
+    "company_id": {
+      "type": "string"
+    },
+    "status": {
+      "type": ["null", "string"]
+    },
+    "debit_date": {
+      "type": ["null", "string"],
+      "format": "date"
+    },
+    "debit_amount": {
+      "type": ["null", "number"]
+    },
+    "reference_number": {
+      "type": ["null", "string"]
+    },
+    "balance": {
+      "type": ["null", "number"]
+    },
+    "paid_amount": {
+      "type": ["null", "number"]
+    },
+    "notes": {
+      "type": ["null", "string"]
+    },
+    "currency": {
+      "type": ["null", "string"]    
+    },
+    "debit_tax": {
+      "type": ["null", "number"]
+    },
+    "subtotal": {
+      "type": ["null", "number"]
+    },
+    "custom_fields": {
+      "type": ["null", "object"]
+    },
+    "conversion_rate": {
+      "type": "number"
+    },
+    "reversed_on": {
+      "type": ["null", "string"],
+      "format": "date"
+    },
+    "debit_pdf_url": {
+      "type": ["null", "string"],
+      "format": "uri"
+    },
+    "created_by": {
+      "type": ["null", "string"]
+    },
+    "updated_by": {
+      "type": ["null", "string"]
+    },
+    "created_date": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "updated_date": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "billing_contact": {
+      "type": ["null", "object"]
+    },
+    "shipping_contact": {
+      "type": ["null", "object"]
+    },
+    "debit_lines": {
+      "type": [
+				"null",
+				"array"
+			]
+    },
+    "payments": {
+      "type": [
+				"null",
+				"array"
+			]
+    },
+    "credits": {
+     "type": [
+				"null",
+				"array"
+			]
+    },
```

Comment:
@kunalvashistordway 
can we describe this object further?
for instance we describe the array contents as below

https://github.com/singer-io/tap-ordway/blob/master/tap_ordway/schemas/credits.json#L82-L93

Reply:
```json
"invoices": {
			"type": [
				"null",
				"array"
			],
			"items": {
				"type": [
					"null",
					"object"
				],
				"additionalProperties": true
			}
		}
```

for the above invoices the type of content is defined in the items dict, can you please update the schema accordingly.